modules/statistics/RTPStatsCollector.js

```diff
@@ -386,10 +386,10 @@ StatsCollector.prototype._defineGetStatValueMethod = function(keys) {
     let itemStatByKey;
 
     switch (this._browserType) {
-    case RTCBrowserType.RTC_BROWSER_CHROME:
-    case RTCBrowserType.RTC_BROWSER_OPERA:
-    case RTCBrowserType.RTC_BROWSER_NWJS:
-    case RTCBrowserType.RTC_BROWSER_ELECTRON:
+    case browsers.CHROME:
+    case browsers.OPERA:
+    case browsers.NWJS:
+    case browsers.ELECTRON:
```

Comment:
It'd be nice to fix the TODO below -- we now have the tools for it!

Reply:
Do you mean moving the logic from the switch somewhere into the browser module? Or maybe creating a method supportsStatsMethod()?

Reply:
Also I will definitely do it but I'd rather do another PR to avoid complexity in this one.

Reply:
> Do you mean moving the logic from the switch somewhere into the browser module? Or maybe creating a method supportsStatsMethod()?

I don't know whether this use justifies an additional method, and if it does I don't know what the right place for it is (BrowserCapabilities vs kept local to RTPStatsCollector). In any case, it should use browser.isChrome() instead of comparing the name to the constant explicitly.

Reply:
Oh sure! It was looking weird to me when I was refactoring but I didn't figured why :)